app/models/bulkrax/csv_entry.rb

```diff
@@ -49,7 +49,7 @@ def keys_without_numbers(keys)
     end
 
     def key_without_numbers(key)
-      key.sub(/_\d+/, '').sub(/^\d+_/, '')
+      key.sub(/_\d+/, '').sub(/_\d+/, '').sub(/^\d+_/, '')
```

Comment:
the double check is needed to handle `alpha_1_2`.
doing `key.sub(/_\d+_\d+/, '').sub(/^\d+_/, '')` also worked for the above example, but failed for `alpha_1`.

Reply:
can you use gsub instead?

Reply:
ahhhhh. yeah, that works! thanks!